packages/core/src/i18n/i18nUtil.ts

```diff
@@ -123,3 +131,19 @@ export const deriveLabelForUISchemaElement = (uischema: Labelable<boolean>, t: T
   const i18nKey = typeof i18nKeyPrefix === 'string' ? `${i18nKeyPrefix}.label` : stringifiedLabel;
   return t(i18nKey, stringifiedLabel, { uischema: uischema });
 }
+
+export const translateElements = (
+  t: Translator,
+  controlElements: DefaultTranslation [],
```

Comment:
Please rename this to better represent what is expected here: The default translations. Thus, I suggest `defaultTranslations`